tree/ntupleutil/v7/src/RNTupleInspector.cxx

```diff
@@ -228,42 +229,67 @@ const std::vector<ROOT::ENTupleColumnType> ROOT::Experimental::RNTupleInspector:
 void ROOT::Experimental::RNTupleInspector::PrintColumnTypeInfo(ENTupleInspectorPrintFormat format, std::ostream &output)
 {
    struct ColumnTypeInfo {
-      std::uint32_t count;
-      std::uint64_t nElems, compressedSize, uncompressedSize;
+      std::uint64_t nElems = 0, compressedSize = 0, uncompressedSize = 0;
+      std::uint64_t nPages = 0;
```

Comment:
With the added zero-initialization, I would suggest to put each variable on a separate line.